lerna.json

```diff
@@ -6,14 +6,7 @@
     "exact": true,
     "command": {
         "bootstrap": {
-            "npmClientArgs": [
-                "--no-audit",
-                "--no-optional",
-                "--loglevel error",
-                "--no-progress",
-                "--unsafe-perm",
-                "--prefer-offline"
-            ]
+            "npmClientArgs": ["--no-audit", "--no-optional", "--loglevel error", "--no-progress", "--unsafe-perm"]
```

Comment:
@Yeti-or @neretin-trike @kayman233 

Временно убираем `--prefer-offline`.

> If you run npm install --prefer-offline it instead uses whatever is in the npm cache that matches the requested version and only if there is nothing in the local cache, it finds and downloads the latest version

В нашем случае почему не  было шага - "it finds and downloads the latest version"

А в релизе упало с ошибкой

```bash
lerna ERR! npm install --no-save --no-audit --no-optional --loglevel error --no-progress --unsafe-perm --prefer-offline exited 1 in '@salutejs/plasma-website'
lerna ERR! npm install --no-save --no-audit --no-optional --loglevel error --no-progress --unsafe-perm --prefer-offline stderr:
npm WARN config optional Use `--omit=optional` to exclude optional dependencies, or
npm WARN config `--include=optional` to include them.
npm WARN config 
npm WARN config     Default value does install optional deps unless otherwise omitted.
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @salutejs/plasma-tokens-b2c@0.34.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Предлагаю время разбирательства почему так, не использовать этот флаг. 

https://github.com/salute-developers/plasma/actions/runs/5961412423/job/16171607020

Reply:
я не понял при чем тут `prefer-offline` если упали optional Deps

Reply:
странная хрень. Причём тригернулась только на plasma-website, который использует иконки

Reply:
@Yeti-or @neretin-trike 

Больше того локально я смогу такую ошибку поймать. 

Ставил на чистую и один-в-один как в CI.  

> просто перезапуск не помогал?

Делал, не помогло.

-----

В #669 тоже такие проблемы - https://github.com/salute-developers/plasma/actions/runs/5962074304/job/16172535924?pr=669

----

> я не понял при чем тут prefer-offline если упали optional Deps

prefer-offline пытается найти в локальном cache и только потом идти скачивать. 

И почему ошибка я пока не понял. (